titan/output.py

```diff
@@ -153,11 +156,12 @@ def get_stats(
             agent_feature = getattr(a, reportable.name)
             agent_feature.set_stats(stats_item, time)
 
-    for a in deaths:
-        stats_item = get_stats_item(stats, attrs, a)
-        add_agent_to_stats(stats_item, "deaths")
-        if a.hiv.active:  # type: ignore[attr-defined]
-            add_agent_to_stats(stats_item, "deaths_hiv")
+    for t in exits:
+        for a in exits[t]:
+            stats_item = get_stats_item(stats, attrs, a)
+            add_agent_to_stats(stats_item, t)
+            if a.hiv.active and t == "death":  # type: ignore[attr-defined]
+                add_agent_to_stats(stats_item, "deaths_hiv")
```

Comment:
should this be in hiv? I think it's because we only have the deaths here.

Reply:
Yeah, this is the only place we have deaths, so we can't get this info in hiv without passing death in.